playwright/tests/fixtures/baseTest.ts

```diff
@@ -29,6 +29,7 @@ const runBrowserTest = async (workspace: string, take: (r: Page) => Promise<void
   await page.goto(`http://localhost:3000/?folder=${tmpWorkspace}`);
   await initialize(page);
   await take(page);
+  await page.goto('http://localhost:3000');
   await browser.close();
```

Comment:
browser.close() does not close the socket connections in case of chromium.

Reply:
not good. what do we need to do?

Reply:
we need the goto